### DISCUSSION TITLE
Please Approval of WalletConnect registry

### DISCUSSION THREAD

**Discussion by Thinhnguyen185** [WalletConnect/walletconnect-monorepo#1202] [Developer Support]
Hi there!
We have submitted an application to register our wallet "3S Wallet"
Would you be so kind as to approve the application so that our users can access WallectConnect as soon as possible.
Of this present moment, my App "3S wallet" in live on app store and ch play 
CHPlay: https://play.google.com/store/apps/details?id=network.bho.wallet
IOS: https://apps.apple.com/vn/app/3s-wallet/id1622316272
Thanks in advance.

**Comment by hackyguru** [comment]
3S wallet was reviewed last week, thank you!
